api-controllers/routes/manage.js

```diff
@@ -33,6 +36,7 @@ router.use(expressSession({
 router.get('/auth/cf', controller.handler('redirectToAuthorizationServer'));
 router.get('/auth/cf/callback', controller.handler('handleAuthorizationResponse'));
 router.use('/instances/:service_id/:plan_id/:instance_id', instanceRouter);
+router.use('/manage/dashboards/:instance_type/instances/:instance_id', dashboardRouter);
```

Comment:
The URL is constructed as below `manage/dashboards/${plan.manager.name}/instances/${instanceId}` - so shouldnt the URL param `instance_type` be named as `operator` ? now that we have named all managers as operator? It does reflect instance type, i am only commenting it based on how this value is set i.e. from attrib `plan.manager.name` - which reflects  operator name.

Reply:
backup/restore also operator. So, kept as **instance_type**. Could be **provisioner** or **provisioner_type**. Also constant is **CONST.INSTANCE_TYPE.DIRECTOR**. That's why kept **instance_type**.

Reply:
Ok.